interpreter/cling/lib/Interpreter/IncrementalExecutor.h

```diff
@@ -116,7 +116,8 @@ namespace cling {
     ///\brief Function registered via __cxa_atexit, atexit, or one of
     /// it's C++ overloads that should be run when a module is unloaded.
     ///
-    typedef utils::OrderedMap<llvm::Module*, std::vector<CXAAtExitElement>>
+    typedef utils::OrderedMap<std::shared_ptr<llvm::Module>,
```

Comment:
Should probably be a non-owning `weak_ptr`.

Reply:
Added a FIXME.